docs/agile-methodologies.md

```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
+10. Team and Product Owner point out if this is the best plan they can make given what they know right now.
+11. Get back to work.
+
+
+### **Grooming Session**
+Backlog grooming, also referred to as backlog refinement or story time, is a recurring event for agile product development teams. The primary purpose of a backlog grooming session is to ensure the next few sprints worth of user stories in the product backlog are prepared for sprint planning. Regular backlog grooming sessions also help ensure the right stories are prioritized.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/UpEBfS9SZGM" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Who is involved?
+
+- **Product Owner:** Is tasked with facilitating backlog refinement sessions. However, that doesn’t mean they are solely responsible for holding backlog grooming sessions.
+- **Team Members:** These events are meant to be collaborative. That means the entire cross-functional team should be represented at refinement sessions.
+- **QA representatives:** You need the combined expertise of the various functions on your team to effectively flesh out your user stories.
+
+### Grooming steps
+
+1. Removing user stories that no longer appear relevant.
+2. Creating new user stories in response to newly discovered needs.
+3. Re-assessing the relative priority of stories.
+4. Assigning estimates to stories which have yet to receive one
+correcting estimates in light of newly discovered information.
+5. Splitting user stories which are high priority.
+6. Get back to work.
+
+### **Stand-up Session**
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last approximately 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
+
+True to its name, all participants in stand-ups usually remain standing to keep the meetings short and on-topic. However, virtual stand-ups are also possible. 
+
+In Agile project management, daily stand-up meetings are essential. These meetings allow project members to share critical information, openly discuss issues, and hold themselves and each other accountable.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/YHMw3CL5cXE" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Benefits of daily Agile stand-ups
+
+- Daily stand-ups allow team members to work collaboratively toward project goals. 
+- Daily stand-up meetings are important for keeping Agile teams focused and on-task while providing quick, project-level updates to the rest of the team. 
+- The Agile methodology is all about versatility and flexibility, it’s important to make tweaks and improvements to your meetings to fit your team’s needs. 
+- Your daily stand-up should inform and draw out issues so that you can keep your project on track and get ahead of issues before they pop up.
+
+
+### **Retrospective Session** 
```

Comment:
does using bold `**` tags  inside the  H3 tag `###` make a difference in styles?

Reply:
This is the difference.
![image](https://user-images.githubusercontent.com/31549144/105867132-57693300-5fba-11eb-95be-1442c6298491.png)